src/content/plugins/profiling-plugin.md

```diff
@@ -27,8 +27,8 @@ new webpack.debug.ProfilingPlugin({
 
 In order to view the profile file:
 
-- Run webpack with `ProfilingPlugin`.
-- Go to Chrome, open the `Profile Tab`.
-- Drag and drop generated file (`events.json` by default) into the profiler.
+1. Run webpack with `ProfilingPlugin`.
+2. Go to Chrome, open devtools, and go to the the `Performance` tab (formerly `Timeline`).
```

Comment:
Can you change this to "[DevTools](https://developers.google.com/web/tools/chrome-devtools/)"?

Reply:
Done